.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This file should not be part of this PR. Please remove. You can even add this to the gitignore to make sure nobody will push these files anymore